pxfish/pyfish.py

```diff
@@ -256,34 +257,41 @@ def push(aq, directory, files_to_write):
 def main():
     parser = argparse.ArgumentParser(description="Pull or Push files from/to Aquarium")
     parser.add_argument("action", choices=["push", "pull"], help="Indicate if you want to push files or pull them.")  
-    parser.add_argument("-d", "--directory", required=True, help="The name of the directory where the files should be written. If the directory does not already exist, it will be created")
+    parser.add_argument("-d", "--directory", help="The name of the directory where the files should be written. If the directory does not already exist, it will be created")
     parser.add_argument("-c", "--category", help="The Aquarium category where the OperationType or Library is located")
 
     group = parser.add_mutually_exclusive_group()
     group.add_argument("-l", "--library", help="the Library you want to pull")
     group.add_argument("-o", "--operation_type", help="the Operation Type you want to pull")
-    
+   
     args = parser.parse_args()
     
+    if not args.directory:
+        args.directory = os.getcwd()
+
     aq = open_aquarium_session()
 
     path = os.path.normpath(args.directory)
     makedirectory(path)
 
-    if args.library: 
-        get_library(aq, path, args.category, args.library) 
-    elif args.operation_type:
-        get_operation_type(aq, path, args.category, args.operation_type)
-    elif args.category: 
-        get_category_optypes_and_libraries(aq, path, args.category)
-    else:
-        get_all_optypes_and_libraries(aq, path)
-
-    if args.action == 'push':
-        if args.library:
-            push_library(aq, args.directory, args.category, args.library)
+    if args.action == 'pull':
+        if args.library: 
+            get_library(aq, path, args.category, args.library) 
+        elif args.operation_type:
+            get_operation_type(aq, path, args.category, args.operation_type)
+        elif args.category: 
+            get_category(aq, path, args.category)
         else:
+            get_all_optypes_and_libraries(aq, path)
+    elif args.action == 'push':
```

Comment:
you could do

```python
if args.action != 'push':
    logging.warning("You must indicate whether you would like to 'push' or 'pull'")
    return
```

and then rest would be for push, but follow similar pattern to code for pull